### PR TITLE
feat(boards): update boards to a tile style layout

### PR DESCRIPTION
**Pull request by benox3**

### CODE REVIEW

File: src/components/Boards/components/BoardTile/BoardTile.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+import Avatar from '@material-ui/core/Avatar';
+import CardActionArea from '@material-ui/core/CardActionArea';
+import Card from '@material-ui/core/Card';
+
+interface Props {
+  id: string;
+  name: string;
+}
+
+export default function DefaultListItem(props: Props) {
```
Comment (benox3) [inline src/components/Boards/components/BoardTile/BoardTile.tsx]: - [ ] Update function name
- [ ] Add Story

File: src/components/Boards/components/BoardTile/BoardTile.styles.tsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import { styled } from '@material-ui/core/styles';
```
Comment (benox3) [inline src/components/Boards/components/BoardTile/BoardTile.styles.tsx]: - [ ] Remove this file. We don't need it anymore.

File: src/components/Boards/Boards.tsx [hunk 3]
```diff
@@ -14,18 +14,23 @@ interface Props {
...
+    <>
+      <Box m={theme.spacing(0.5)}>
+        <Typography variant="h4" component="h1">
+          Boards
+        </Typography>
+      </Box>
+      <Divider />
+      <Box m={theme.spacing(0.5)}>
+        <Grid container spacing={4}>
+          {props.boards.map(board => {
```
Comment (remarkablemark) [inline src/components/Boards/Boards.tsx]: No need for change, but could refactor to following:
```tsx
props.boards.map(board => <BoardTile key={board.id} {...board} />)
```

File: src/components/Boards/components/BoardTile/BoardTile.tsx [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+  return (
+    <Grid item xs={6} sm={3}>
```
Comment (remarkablemark) [inline src/components/Boards/components/BoardTile/BoardTile.tsx]: Question: How does it look good on larger screens?

File: src/components/Boards/components/BoardTile/BoardTile.tsx [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+      <Card>
+        <CardActionArea>
+          <CardHeader
+            avatar={
+              <Avatar aria-label="recipe">
+                {props.name ? props.name[0].toUpperCase() : ''}
```
Comment (remarkablemark) [inline src/components/Boards/components/BoardTile/BoardTile.tsx]: Question: Do we allow `name` to start with a space?

File: src/components/Boards/components/BoardTile/BoardTile.tsx (hunk 4 above)
Comment (benox3) [reply src/components/Boards/components/BoardTile/BoardTile.tsx]: Looks ok -- little wide. We should probably wrap the container at some point. We can do polish later.

File: src/components/Boards/components/BoardTile/BoardTile.tsx (hunk 5 above)
Comment (benox3) [reply src/components/Boards/components/BoardTile/BoardTile.tsx]: We're not sanitizing anything currently. We'll have to trim on the way in

Comment (remarkablemark) [reply src/components/Boards/components/BoardTile/BoardTile.tsx]: Gotcha, we can ticket this and handle it in the future.

File: src/components/Boards/components/BoardTile/BoardTile.tsx (hunk 4 above)
Comment (remarkablemark) [reply src/components/Boards/components/BoardTile/BoardTile.tsx]: Nice!

**Pull request merged**